streamlit-javascript/app/setup_script.sql

```diff
@@ -0,0 +1,31 @@
+-- This is the setup script that runs while installing a Snowflake Native App in a consumer account.
+-- For more information on how to create setup file, visit https://docs.snowflake.com/en/developer-guide/native-apps/creating-setup-script
+
+-- A general guideline to building this script looks like:
+-- 1. Create application roles
+CREATE APPLICATION ROLE IF NOT EXISTS app_public;
+
+-- 2. Create a versioned schema to hold those UDFs/Stored Procedures
+CREATE OR ALTER VERSIONED SCHEMA core;
+GRANT USAGE ON SCHEMA core TO APPLICATION ROLE app_public;
+
+-- 3. Create UDFs and Stored Procedures using the JavaScript handler. 
+CREATE or REPLACE FUNCTION core.add(num1 DOUBLE, num2 DOUBLE)
```

Comment:
```suggestion
CREATE or REPLACE FUNCTION core.add(NUM1 DOUBLE, NUM2 DOUBLE)
```
As a matter of style I think it makes sense to keep the case the same (I understand we're going between two different case "systems" here)